docs/source/models/gbd2017/causes/latent_tuberculosis/index.rst

```diff
@@ -4,6 +4,119 @@
 Latent Tuberculosis
 ===================
 
-.. todo:
+TB Overview
+-----------
 
-   all of this
+TB definitions
+++++++++++++++
+Tuberculosis (TB) is a disease caused by the bacteria Mycobacterium
+tuberculosis. It most commonly affects the lungs, although it can affect
+other parts of the body. TB can be fatal if not recognized and treated.
+It also can spread from person to person to infect others. Anyone who 
+inhales the airborne droplets that contains TB bacteria is called a "contact".
+A contact can be someone who living with the person who has infectious TB,
+like a family member, friend, or coworker. [UpToDate]_
+
+What is latent TB?
+++++++++++++++++++
+Latent tuberculosis infection (LTBI) is defined as a state of persistent
+immune response to stimulation by Mycobacterium tuberculosis antigens with
+no evidence of clinically manifest active TB. [WHO]_
+
+What is active TB?
+++++++++++++++++++
+Active tuberculosis, also known as reactivation TB. It occurs when individual's
+immune system becomes weakened and is no longer able to contain the latent
+bacteria; then the TB bacteria become "active", overwhelm the immune system and
+cause a person to become ill. [UpToDate]_ In GBD, active TB consist of 6 child
+causes: Drug-susceptible TB, Multidrug-resistant TB, Extensively drug-resistant
+TB, HIV/AIDS - Drug-susceptible TB, HIV/AIDS - Multidrug-resistant TB, and
+HIV/AIDS - Extensively drug-resistant TB.
+
+TB progression
+++++++++++++++
+Active TB disease developed within 2 years called fast progression, Individuals
+who do not have rapid progression are classified as having slow-progressing latent
+tuberculosis infection. With latent infection, individuals experience no adverse
+health effects and will not transmit Mycobacterium tuberculosis, but they face
+an ongoing risk of developing active tuberculosis through reactivation.
+
+GBD 2017 Modeling Strategy
+++++++++++++++++++++++++++
+GBD 2017 TB model made a refinement in the mortality-to-incidence (MI) ratio
+regression approach upon 2016 model. The SDI was replaced by Healthcare Access
+and Quality index (HAQI) in the MI ratio regression. First, GBD estimated
+risk-weighted prevalence of LTBI by demographic characteristics using data from
+population-based tuberculin surveys and cohort studies reporting the risk of
+developing active TB disease as a function of induration size. Next, they
```

Comment:
prior to this point, i think it would be helpful to explain TSTs and what induration size is!

Reply:
Good catch! It’s important to explain the relationship between TST and induration size because the distribution is dependent on sub-population selection.